src/content/deployment/flavors-ios.md

```diff
@@ -0,0 +1,471 @@
+
+1.  Run the scheme for your app
+    (**Product > Run**).
+
+## Customize configurations
+
+After you've added Xcode schemes, you can customize them for
+your iOS app. You can also use these steps to configure a
+macOS project by replacing any reference to `iOS` with
+`macOS`.
+
+### Create a distinct app display name
+
+If you have multiple schemes, a distinct app name can
+quickly identify which scheme your deployed app is using.
+
+<img src="/assets/images/docs/flavors/flavors-ios-app-names.png" alt="Rename a Flutter flavor" width="50%">
+
+The following steps show how to add distinct app display
+names in Xcode for two schemes called `staging` and
+`production` in a project called `flavors_example`.
+
+1.  Create user-defined settings in Xcode:
+
+    * Open the **project navigator**
+      (**View > Navigators > Project**).
+
+    * In the **project navigator**, at the top, select
+      **Runner**.
+
+    * In the main window under **TARGETS**, select
+      **Runner**.
+
+    * Open the **Build Settings** tab.
+
+    * To the left of the Basic tab, click **+** and select
+      **Add User-Defined Setting**.
+
+    * Create a setting named `APP_DISPLAY_NAME`.
+
+    * Expand the **APP_DISPLAY_NAME** setting.
+
+    * Assign the following values to the following keys:
+
+      * **Debug-production**: `Flavors prod`
+      * **Debug-staging**: `Flavors staging`
+      * **Profile-production**: `Flavors prod`
+      * **Profile-staging**: `Flavors staging`
+      * **Release-production**: `Flavors prod`
+      * **Release-staging**: `Flavors staging`
+
+1.  Update `Info.plist` in Xcode:
+
+    * In the project navigator, select
+      **Runner > Runner > Info** to open
+      `flavor_test/ios/Runner/Info.plist`.
+
+    * Under **Information Property List**, find the
+      following key and update the value for it:
+
+      * **Key**: `Bundle display name`
+      * **Value**: `$(APP_DISPLAY_NAME)`
+
+1.  Launch the app for each scheme (`staging`, `production`)
+    and check to make sure that the app display name has
+    changed for each. To launch a scheme, see the steps in
+    [Launch an Xcode scheme][].
+
+[Launch an Xcode scheme]: #launch-an-xcode-scheme
+
+### Create distinct icons
+
+If you have multiple schemes, a distinct icon for each
+configuration can help you quickly identify which scheme
+your deployed app is using.
+
+<img src="/assets/images/docs/flavors/flavors-ios-icons.png" alt="Rename a Flutter flavor" width="50%">
+
+The following steps show how to add a distinct icon in
+Xcode for two schemes called `staging` and `production` in
+an iOS project called `flavors_example`.
+
+1.  Prepare your icons:
+
+    * Design your staging icon and production icon in the
+      design tool of your choice.
+
+    * Generate versions of the staging icon and production
+      icon in the sizes that you need. Save them in
+      PNG format.
+
+      Note: You can use a tool like [App Icon Generator][]
```

Comment:
Should this be an actual `:::note` ?

Reply:
Yes it should. Thanks for catching this. Updated here and in one more place.